example/code-snippets/main.jsx

```diff
@@ -36,10 +36,10 @@ const dateRanges = [
   },
 ];
 
-var DatePicker = React.createClass({
+export var DatePicker = React.createClass({
```

Comment:
Getting a linting error for an unused var, as this is just a code snippet. I _think_ we probably want to lint this file but not sure what the best way to deal with this linting error. Some possible solutions:

1. Add an `/* eslint-disable no-unused-vars */`
2. Throw an `export` in front of `var DatePicker = React.createClass({` as I've done here.